src/components/scenes/TransactionsExportScene.js

```diff
@@ -210,7 +214,7 @@ class TransactionsExportSceneComponent extends React.PureComponent<Props, State>
     if (isExportCsv) {
       files.push({
         contents: csvFile,
-        mimeType: 'text/csv',
+        mimeType: 'text/comma-separated-values',
```

Comment:
Please make sure this change works on iOS. If so this PR is approved and can be merged